src/gardener/gardener-tree.js

```diff
@@ -296,6 +326,78 @@ function plantTooltip (element) {
   return list.join('\n')
 }
 
+async function backupBuckets ({ parent: landscape }) {
+  const backupBucket = new BackupBucketClient(landscape.kubeconfig)
+  const backupBuckets = await backupBucket.list()
+  return _.map(backupBuckets, backupBucket => {
+    return toBackupBucketTreeNode(landscape, backupBucket)
+  })
+}
+
+function toBackupBucketTreeNode (landscape, backupBucket) {
+  const name = _.get(backupBucket, 'metadata.name')
+  return {
+    nodeType: nodeType.NODE_TYPE_BACKUP_BUCKET,
+    kindPlural: 'backupbuckets',
+    name,
+    landscape,
+    seed: _.get(backupBucket, 'spec.seed'),
+    region: _.get(backupBucket, 'spec.region'),
```

Comment:
```suggestion
    region: _.get(backupBucket, 'spec.provider.region'),
```

Reply:
Thanks for pointing that out. I implemented it initally And forgot to change that up in PR. I'll do the necessary changes.